cli/operator/keys/create.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	OperatorKeyFolder = "operator_keys"
+	OperatorKeyFolder = ".eigenlayer/operator_keys"
```

Comment:
Rename to `OperatorKeyFolderRelToHome` or something?

Reply:
`OperatorKeystoreSubFolder` it is